loader.js

```diff
@@ -161,12 +161,12 @@ module.exports = function(source) {
 
       return newDoc;
     }
-
+    
```

Comment:
I can imagine, that this line produces a mental dissonance on the apollo team. They don't know exactly why they can't merge this, they just know that they cannot merge it. 

@dobesv removing this line will make wonders